src/models/Polygon.js

```diff
@@ -7,7 +7,18 @@ const options = {
     polygonSides: {
       title: 'Number of sides',
       min: 3
-    }
+    },
+    roundCorners: {
+      title: 'Round Corners',
```

Comment:
For consistency, I would not capitalize more than the first word in the labels... 'Round corners' and 'Round fraction'